src/types/index.tsx

```diff
@@ -18,12 +18,12 @@ export type DynamicField = Field & {
 };
 
 export type Student = {
+  [DefaultFieldKey.DATE_OF_BIRTH]: Date | null;
```

Comment:
sigh I'm unable to actually submit the form because the validator fails - the backend expects the date to be submitted [as a string](https://www.django-rest-framework.org/api-guide/fields/#datefield) so we'll wanna submit a string in YYYY-MM-DD format or null 😖

Reply:
fixed